### ISSUE TITLE
L2 regularization for dense layer

### ISSUE THREAD

**Issue by ThomasPerrais** [ThomasPerrais/tf_classification#2]
it sometimes seems that the network overfit on the training data as the predictions scores are often really big values. It might be because of a lack of regularization (despite dropout ?) in the dense layer?
